hazelcast/src/main/java/com/hazelcast/instance/AddressPicker.java

```diff
@@ -55,6 +55,8 @@
 
     Map<EndpointQualifier, Address> getPublicAddressMap();
 
+    Map<EndpointQualifier, Address> getBindAddressMap();
```

Comment:
Could you add the javadoc (and also fill the missing one in the `getPublicAddressMap`?

Reply:
Added related javadocs.